community/inventory_service_tags_german.txt

```diff
@@ -1,9 +1,18 @@
+1. Einzigartig | %ITEMNAME%
```

Comment:
Before I put this in (because there's no removing it, only changing it) is the pipe character intended? This would be used by the Steam Community website in a case where there's an item called something like "Silly Hat" and the item has custom tags from a generator to show the name as Unique Silly Hat. Is `Einzigartig | (the german word(s) for silly hat)` the correct string to display in this case?

Reply:
Yes it is intended, because I can't form a nice generic way to write it out like in english.
English:
* Unique Hat
* Unique Sword
* Unique Shoes
German:
* EinzigartigER Hut
* EinzigartigES Schwert
* EinzigartigE Schuhe

So it's only meant to seperate the rarity visualy from the itemname.
I thought about a colon, but it doesn't seem to fit.

Maybe `%ITEMNAME% (Einzigartig)` or `%ITEMNAME%, Einzigartig` is better, I'm not sure.

If I could just have a real visual representation how it i used, that would help.
I looked into the steam-inventory but can't find a place where it looks like how you described it.

Reply:
Unfortunately, the part of the admin panel that gives examples is not translated:

![image](https://user-images.githubusercontent.com/4257305/180624994-79daff2f-a594-4084-acff-c3d7913886c3.png)

Reply:
Oh, the documentation for accessory items seems to be translated: https://partner.steamgames.com/doc/features/inventory/accessories?l=german

Reply:
Ok, yea, no, I think I really stick to that format, thanks for looking it up.